nssp_demo/postprocess_state_forecast.R

```diff
@@ -227,26 +244,55 @@ make_forecast_figs <- function(model_run_dir,
       values_to = ".value"
     )
 
+  arrow::write_parquet(
+    posterior_predictive_samples,
+    path(model_run_dir, "forecast_samples",
+      ext = "parquet"
+    )
+  )
+
   posterior_predictive_ci <-
     posterior_predictive_samples |>
     select(date, disease, .value) |>
     group_by(date, disease) |>
     median_qi(.width = c(0.5, 0.8, 0.95))
 
 
+  arrow::write_parquet(
+    posterior_predictive_ci,
+    path(model_run_dir, "forecast_ci",
+      ext = "parquet"
+    )
+  )
+
+
   all_forecast_plots <- map(
-    set_names(unique(dat$disease)),
+    set_names(unique(combined_dat$disease)),
     ~ make_one_forecast_fig(
       .x,
-      dat,
+      combined_dat,
       last_training_date,
-      last_data_date,
+      data_vintage_date,
       posterior_predictive_ci,
       state_abb
     )
   )
 
-  return(all_forecast_plots)
+  all_forecast_plots_log <- map(
+    all_forecast_plots,
+    \(x) x + scale_y_continuous(transform = "log10")
```

Comment:
This gets rid of all of the other formatting for the y-axis.

Reply:
Should be fixed by using `coord_trans` instead in 156c4ec